src/main/java/mServer/crawler/sender/wdr/WdrLetterPageDeserializer.java

```diff
@@ -0,0 +1,77 @@
+package mServer.crawler.sender.wdr;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class WdrLetterPageDeserializer extends WdrDeserializerBase {
+    
+    private static final String QUERY_SENDUNG = "ul.list > li > a";
+    private static final String QUERY_TITLE = "span";
+    private static final String QUERY_URL_TYPE = "strong";
+    
+    public List<WdrSendungDto> deserialize(Document document) {
+        List<WdrSendungDto> list = new ArrayList<>();
+        
+        Elements sendungenElement = document.select(QUERY_SENDUNG);
+        sendungenElement.forEach(sendungElement -> {
+            String url = getUrl(sendungElement);
+                    
+            if(!url.isEmpty()) {
+                WdrSendungDto dto = new WdrSendungDto();
+                dto.setTheme(getTheme(sendungElement));
+
+                switch(getUrlType(sendungElement)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)